main.py

```diff
@@ -51,14 +52,14 @@ def append_testcases(prefix, testsuite):
     test_count = 0
     failure_count = 0
 
-    with open(prefix + '_requests.csv', mode='r') as csv_file:
+    with open(os.getcwd() + '\\' + prefix + '_stats.csv', mode='r') as csv_file:
```

Comment:
Same here - Let's create a new local, `csv_file_path`, and use `os.path.join` to combine the directory and the path.